config/connect-mirror-source.properties

```diff
@@ -0,0 +1,31 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name=local-mirror-source
+connector.class=org.apache.kafka.connect.mirror.MirrorSourceConnector
```

Comment:
Yes, MirrorMaker fills this in for you normally. This config file is provided just for running MirrorSourceConnector in "standalone mode" as follows:

   ./bin/connect-standalone.sh config/connect-standalone.properties config/connect-mirror-source.properties

i.e. without the top-level MM2 driver doing the work for you. Organizations that already have a Connect-as-a-Service cluster will find this useful as well, as they may wish to leverage their existing cluster and just configure it to run the MM2 connectors.

Reply:
I have been testing this branch and used the connector.class option noted above in my configuration.  I noticed that records were being copied 3X instead of the expected 1.  Debugging through the source, apparently that configuration will override the connector name that is setup during MirrorMakerConfig.connectorBaseConfig.  Since MIrrorMaker attempts to setup the Source, Health, and Checkpoint connectors, I actually ended up with 3 source connectors.

Reply:
@jeremy-l-ford Funny problem! I think this, along with others' experiences here, indicates I need to remove this sample configuration altogether, and just provide a top-level configuration file like in the "quick start" above. The connector configuration is just confusing. Moreover, it doesn't really demonstrate anything beyond the existing generic connect-standalone.properties file.

I'll remove this file, thanks.